es-es/crystal-es.html.markdown

```diff
@@ -0,0 +1,542 @@
+---
+idioma: cristal
```

Comment:
I don't think it is valid to translate the language name. Check other translations, Python is not "Pitón", Dart is not "dardo", Ruby is not "Rubí"